packages/gatsby-link/src/index.js

```diff
@@ -3,7 +3,7 @@ import { Link, NavLink } from "react-router-dom"
 import PropTypes from "prop-types"
 
 let pathPrefix = ``
-if (__PREFIX_PATHS__) {
+if (typeof __PREFIX_PATHS__ !== "undefined" && __PREFIX_PATHS__) {
```

Comment:
I agree Lodash is generally nicer but as gatsby-link is used on *every* Gatsby site out there, we want to add as little code as possible so the "raw" check in this PR is better.